examples/pytorch/wan/test_and_benchmark.py

```
@@ -0,0 +1,1120 @@
+                output = model(
+                    hidden_states, timestep, encoder_hidden_states, return_dict=False
+                )
+
+            if output[0].shape != hidden_states.shape:
+                print(f"  {backend}: FAILED (shape mismatch)")
+                all_passed = False
+            else:
+                print(f"  {backend}: PASSED (FlashInferLinear: {fi_count})")
+            del model
+            torch.cuda.empty_cache()
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)